tools/github-promote-release.ts

```diff
@@ -25,26 +25,28 @@ const getReleaseId = async ({ deliverinoPrivateKey, owner, repo, platform }: Opt
   const result = releases.data.find(release => release.tag_name.includes(platform))
   if (result && result.prerelease) {
     console.log('Unset prerelease tag of ', result.tag_name)
-    return result.id
+    return { id: result.id, body: result.body }
```

Comment:
:upside_down_face: I think it would be easier to just return the whole release:
```suggestion
    return result
```